metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -265,3 +265,14 @@ spec:
         floor((ocs_storage_provider_operator_version>0)/1000) - ignoring(storage_consumer_name) group_right() floor((ocs_storage_client_operator_version>0)/1000) > 1 or floor((ocs_storage_client_operator_version>0)/1000) - ignoring(storage_consumer_name) group_left() floor((ocs_storage_provider_operator_version>0)/1000) >= 1
       labels:
         severity: critical
+  - name: ceph-perf-warnings.rules
+    rules:
+    - alert: OSDHighCPULoad
+      annotations:
+        description: High CPU usage in the OSD container on node {{ $labels.pod }}. Please create more OSDs to increase performance
+        message: High CPU usage detected in OSD container on node {{ $labels.pod}}.
+        severity_level: warning
+      expr: "pod:container_cpu_usage:sum{pod=~\"rook-ceph-osd-.*\"} > 0.35 \n"
```

Comment:
why 0.35?

Reply:
i've set it to 35%, because in all of my tests
the OSD cpu usage, hovered around 20%ish. in very very overloaded environments, the osds steadly increased cpu usage till 30-40%

the throughput and latency was very low during the overloaded environment, only increased when i deployed a larger cluster (also resulting in lower CPU usage)